net/sscep/Makefile

```
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sscep
+PKG_VERSION:=0.10.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v${PKG_VERSION}.tar.gz
+PKG_SOURCE_URL:=https://github.com/certnanny/sscep/archive/
+PKG_HASH:=489cc8e093986776eb3f15082bf766778f707176f3cd604bf0ef1008da06b8e5
+
+PKG_MAINTAINER:=John Marrett <johnf@zioncluster.ca>
+PKG_LICENSE:=
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/sscep
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+libopenssl +openssl-util
+  TITLE:=A Simple SCEP (Simple Certificate Enrollment Protocol) client
+  URL:=https://github.com/certnanny/sscep/
+endef
+
+define Package/sscep/description
+  sscep is a simple SCEP client published by the CertNanny project.
+
+  It can be used to retrieve CA certificates and enroll devices with a compatible SCEP server.
```

```suggestion
  It can be used to retrieve CA certificates and enroll devices with
  a compatible SCEP server.
```

wrap new line for every 70 chars